frame/pablo/src/lib.rs

```diff
@@ -254,7 +268,8 @@ pub mod pallet {
 			+ Debug
 			+ Default
 			+ TypeInfo
-			+ Ord;
+			+ Ord
+			+ From<CurrencyId>;
```

Comment:
I think where headed to wards interfaces

like 
trait SpecialAssets {
    fn get_local_native() -> 
    fn get_relay_native() ->
    fn get_protocol_native(pallet_id) -> Option<AssetId>
}

Reply:
Good point, Though may be we should wait for a few more use cases before doing such refactorings? Specially the use here is to set rewarded asset types, which can be much more than just what you list out here.